index.html

```diff
@@ -0,0 +1,33 @@
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+  <main class="about-me">
+  <nav>
+  <header class="logo">
+  <h3>Welcome</h3>
+  <a href="" class="menu"><i class="fas fa-bars"></i></a>
+  </header>
+  </nav>
```

Comment:
Kindly make the `header` first then `nav`, so it will be like:
```
<header>
   <nav>
        your code...............
   </nav>
</header> 
```
and put it outside the main tag for semantic purposes.